plugin/src/main/java/org/opensearch/ml/rest/RestMLGetConnectorAction.java

```diff
@@ -59,7 +69,7 @@ public RestChannelConsumer prepareRequest(RestRequest request, NodeClient client
     MLConnectorGetRequest getRequest(RestRequest request) throws IOException {
         String connectorId = getParameterId(request, PARAMETER_CONNECTOR_ID);
         boolean returnContent = returnContent(request);
-
-        return new MLConnectorGetRequest(connectorId, returnContent);
+        String tenantId = getTenantID(clusterService, settings, request);
```

Comment:
I see only this get connector API changed for checking tenant id. Will you add checking for other APIs in a new PR ?